pollination/alias/inputs/simulation.py

```diff
@@ -0,0 +1,20 @@
+from pollination_dsl.alias import InputAlias
+from queenbee.io.common import IOAliasHandler
+
+
+"""Alias for inputs that expect a simulation parameter .json file as the recipe input."""
+ddy_input = [
```

Comment:
I assume this name should change to something else like energy_simulation_parameters.